lib/ace/mode/papeeria_latex_highlight_rules.js

```diff
@@ -0,0 +1,66 @@
+define(function(require, exports, module) {
+"use strict";
+
+var oop = require("../lib/oop");
+var TextHighlightRules = require("./text_highlight_rules").TextHighlightRules;
+
+var PapeeriaLatexHighlightRules = function() {  
```

Comment:
No, no.
We have a completely different state machine here.

New file -- to keep the old highlight